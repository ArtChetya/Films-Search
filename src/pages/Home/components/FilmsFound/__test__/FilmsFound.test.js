import { shallow } from 'enzyme/build';
import React from 'react';
import { FilmsFound } from '..';

describe('FilmsFound component', () => {
    let films;

    beforeAll(() => {
        films = [
            {
                id: 181790,
                title: 'Journey 3: From the Earth to the Moon',
                tagline: '',
                vote_average: 0,
                vote_count: 7,
                release_date: '2018-12-31',
                poster_path:
                    'https://image.tmdb.org/t/p/w500/98tbNloMBztAVnWpAznKKVUdi2O.jpg',
                overview:
                    'Sean and Hank go on their biggest adventure yet, to the moon.',
                budget: 0,
                revenue: 0,
                genres: ['Action', 'Adventure', 'Fantasy', 'Science Fiction'],
                runtime: null
            },
            {
                id: 371638,
                title: 'The Disaster Artist',
                tagline: 'I did naaaht! Oh, hi Mark',
                vote_average: 7.2,
                vote_count: 808,
                release_date: '2017-11-30',
                poster_path:
                    'https://image.tmdb.org/t/p/w500/jj84nF5vYRD0HUTowBKcrKk8hZP.jpg',
                overview:
                    'An aspiring actor in Hollywood meets an enigmatic stranger by the name of Tommy Wiseau',
                budget: 10000000,
                revenue: 27129350,
                genres: ['Comedy', 'Drama', 'History'],
                runtime: 104
            }
        ];
    });

    it('should be rendered correctly', () => {
        const component = shallow(<FilmsFound films={films} />);
        expect(component).toMatchSnapshot();
    });
});
