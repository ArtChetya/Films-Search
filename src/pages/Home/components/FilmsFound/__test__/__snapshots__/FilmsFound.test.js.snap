// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsFound component should be rendered correctly 1`] = `
<ColorText
  fontcolor="#212121"
  variant="subtitle2"
>
  2
   movies found
</ColorText>
`;
